subgraph/src/DandelionVoting.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  StartVote as StartVoteEvent,
+  CastVote as CastVoteEvent,
+  ExecuteVote as ExecuteVoteEvent,
+} from '../generated/templates/DandelionVoting/DandelionVoting'
+import { 
+  getCastEntity, 
+  getProposalEntity, 
+  populateCastDataFromEvent, 
+  populateVotingDataFromContract,
+  populateVotingDataFromEvent
+} from './helpers'
+import { STATUS_EXECUTED } from './statuses'
+import { PROPOSAL_TYPE_DECISION } from './types'
+
+export function handleStartVote(event: StartVoteEvent): void {
+  let proposal = getProposalEntity(event.address, event.params.voteId)
+
+  populateVotingDataFromEvent(proposal, event)
+  populateVotingDataFromContract(proposal, event.address, proposal.number)
+
+  proposal.type = PROPOSAL_TYPE_DECISION
+
+  proposal.save()
+}
+
```

Comment:
should we also have a handler for the config changes?

Reply:
Ah yeah good catch!

Reply:
Done in #77